packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx

```diff
@@ -20,25 +19,25 @@ export const CopyAddressButton: FC<CopyAddressButtonProps> = ({ address, classNa
     if (!address) {
       return
     }
-    copyToClipboard(address, 'Account address copied to clipboard')
+    copyToClipboard(address)
     setCopyButtonClicked(true)
     setTimeout(() => {
       setCopyButtonClicked(false)
     }, 3000)
   }
 
   return (
-    <StyledText
-      as="button"
-      variant={size === 'big' ? 't300' : 't100'}
-      color="colorText"
-      className={className}
-      onClick={handleCopyAddress}
-    >
-      {shortenString(address, 6, 4)}
-      <Tooltip text="Copy account address" placement="top">
+    <StyledTooltip hideOnClick={false} text={copyButtonClicked ? 'Copied!' : 'Copy account address'} placement="top">
```

Comment:
According to designs, this should be `Copy wallet address' now

Reply:
Also, I would change the position of the tooltip if it's possible. Placement should be top left or left 
![image](https://github.com/Joystream/atlas/assets/51168865/a6e09960-6d09-4da6-8739-b56e8fb7a3f2)

![image](https://github.com/Joystream/atlas/assets/51168865/a66748da-26a1-4346-8960-8dbba8c70740)